src/models/Article.php

```diff
@@ -83,8 +88,8 @@ public function rules()
     {
         return [
             [['title', 'text'], 'required'],
```

Comment:
alias/slug should be required i think.

Reply:
I did not want to force usage of category slugs, just add a support. I'm not sure but I think this all should work well for posts that don't have category as well.

Reply:
What makes me curious is if people who have to add news will understand this. I think we should have clear version of how to add news, maybe publishing a fork with all your wishes could be a better solution. Its really just a basic news module.